addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2153,13 +2152,6 @@ var SnippetsMenu = Widget.extend({
             vxml: snippet.dataset.vxml,
         });
     },
-    /**
-     * @private
-     */
-    _onReloadSnippetTemplate: async function (ev) {
-        await this._activateSnippet(false);
-        await this._loadSnippetsTemplates(true);
-    },
```

Comment:
Sorry but... don't remove stuff in stable! ;) Add an explanation in the documentation comment to tell why it was there and why it is unused now. With a TODO so that we know we have to improve that stuff in master without restrictions ;)